src/amo/components/MetadataCard/index.js

```diff
@@ -17,27 +17,30 @@ const MetadataCard = ({ className, metadata }: Props): React.Node => {
 
   return (
     <div className={makeClassName('MetadataCard', className)}>
-      {metadata.map(({ content, title } = {}, index) => {
-        if (content === undefined) {
-          throw new Error('content is required');
-        }
-        if (title === undefined) {
-          throw new Error('title is required');
-        }
-
-        // Empty string and zero values are allowed.
-        const hasContent = content || content === '' || content === 0;
-
-        return (
-          // eslint-disable-next-line react/no-array-index-key
-          <dl className="MetadataCard-list" key={index}>
-            <dd className="MetadataCard-content">
-              {hasContent ? content : <LoadingText />}
-            </dd>
-            <dt className="MetadataCard-title">{title}</dt>
-          </dl>
-        );
-      })}
+      {
+        // eslint-disable-next-line default-param-last
```

Comment:
It seems like there are a couple of things we could do to avoid having to disable this rule:
1. We could assign a default value to `index` as well as to the first argument (which should do nothing as `index` should always have a value).
2. We could change the code to not use destructuring in the `map` arguments, and then we shouldn't need a default value for either argument. This would result in more code inside the `map` function, as we'd have to deal with destructuring in there, but I just feel like the rule is there for a reason, and if we can change our code in a way to not have to disable the rule then that is better.

Having said all that, if you feel it's better to just disable the rule and leave the code as is I am ok with that too.

Reply:
Yeah I wasn't sure what was the right approach here. It's a bit of a similar problem with the reducers, we know this is never going to be called without an `index`, and we don't have a choice in the ordering, so the rule feels a bit pointless here. But I could add a default value to `index` too.